### PR TITLE
feat: Logarithms in expressions DHIS2-9322

### PR DESCRIPTION
**Pull request by jimgrace**

### CODE REVIEW

File: src/commonmark/en/content/user/configure-metadata.md [hunk 1]
```diff
@@ -2156,6 +2156,16 @@ You can use the following functions in an indicator formula:
 <td><p>Returns the least (lowest) value of the expressions given. Can be provided any number of arguments.</p></td>
 </tr>
 <tr class="odd">
+<td><p>log</p></td>
+<td><p>(expression [, base ])</p></td>
+<td><p>Returns the natural logarithm (base e) of the numeric expression. If an integer is given as a second argument, returs the logarithm using that base.</p></td>
```
Comment (enricocolasante) [inline src/commonmark/en/content/user/configure-metadata.md]:
```suggestion
<td><p>Returns the natural logarithm (base e) of the numeric expression. If an integer is given as a second argument, returns the logarithm using that base.</p></td>
```

File: src/commonmark/en/content/user/configure-metadata.md [hunk 2]
```diff
@@ -4907,6 +4917,14 @@ sampled. For example:
         <td><p>least(expression [, expression ...])</p></td>
         <td><p>Returns the least (lowest) value of the expressions given. Can be provided any number of arguments.</p></td>
         </tr>
+        <tr class="odd">
+        <td><p>log(expression [, base ])</p></td>
+        <td><p>Returns the natural logarithm (base e) of the numeric expression. If an integer is given as a second argument, returs the logarithm using that base.</p></td>
```
Comment (enricocolasante) [inline src/commonmark/en/content/user/configure-metadata.md]:
```suggestion
        <td><p>Returns the natural logarithm (base e) of the numeric expression. If an integer is given as a second argument, returns the logarithm using that base.</p></td>
```

File: src/commonmark/en/content/user/configure-programs-in-the-maintenance-app.md [hunk 3]
```diff
@@ -2298,6 +2298,16 @@ functions. The functions can be applied to data elements and attributes:
 <td><p>(expression [, expression ...])</p></td>
 <td><p>Returns the least (lowest) value of the expressions given. Can be provided any number of arguments. Each expression must follow the rules of any program indicator expression (including functions).</p></td>
 </tr>
+<tr class="odd">
+<td><p>log</p></td>
+<td><p>(expression [, base ])</p></td>
+<td><p>Returns the natural logarithm (base e) of the numeric expression. If an integer is given as a second argument, returs the logarithm using that base.</p></td>
```
Comment (enricocolasante) [inline src/commonmark/en/content/user/configure-programs-in-the-maintenance-app.md]:
```suggestion
<td><p>Returns the natural logarithm (base e) of the numeric expression. If an integer is given as a second argument, returns the logarithm using that base.</p></td>
```

**Review by enricocolasante — APPROVED** [review]
One typo to fix

**Comment by jimgrace** [comment]
Thanks @enricocolasante for spotting the typos.

**Pull request merged**